content/develop/data-types/json/indexing_JSON.md

```diff
@@ -9,13 +9,13 @@ categories:
 - oss
 - kubernetes
 - clients
-description: Combine Redis JSON and Search and Query to index and search JSON documents
+description: Combine Redis JSON and the Redis Query Engine to index and search JSON documents
 linkTitle: Index/Search
 title: Index/Search JSON documents
 weight: 2
 ---
 
-In addition to storing JSON documents, you can also index them using the [Search and Query]({{< relref "/develop/interact/search-and-query/" >}}) feature. This enables full-text search capabilities and document retrieval based on their content.
+In addition to storing JSON documents, you can also index them using the [Redis Query Engine]({{< relref "/develop/interact/search-and-query/" >}}) feature. This enables full-text search capabilities and document retrieval based on their content.
 
 To use these features, you must install two modules: RedisJSON and RediSearch. [Redis Stack]({{< relref "/operate/oss_and_stack/" >}}) automatically includes both modules.
```

Comment:
I guess the module is still called RediSearch for now, even though the feature itself is RQE?